addons/wowl/static/src/components/dropdown/dropdown.xml

```diff
@@ -11,7 +11,7 @@
         <t t-slot="default" />
       </span>
     </button>
-    <ul t-if="state.open" class="o_dropdown_menu" t-att-class="props.menuClass">
+    <ul t-if="state.open" t-attf-class="o_dropdown_menu {{ props.align || '' }}" t-att-class="props.menuClass">
```

Comment:
Maybe no need to introduce need prop, we can achieve with menuClass(later I see we already have menuClass prop)